src/extension/src/__main__.py

```diff
@@ -28,14 +29,14 @@
 from extension.src.local_loggers.Logger import Logger
 from extension.src.ProcessHandler import ProcessHandler
 from extension.src.Utility import Utility
-from extension.src.local_loggers.StdOutFileMirror import StdOutFileMirror
 from extension.src.Constants import Constants
 
 
 def main(argv):
     stdout_file_mirror = None
     file_logger = None
-    logger = Logger()
+    telemetry_writer = TelemetryWriter()
+    logger = Logger(telemetry_writer=telemetry_writer)
```

Comment:
This is important, and relates to the ask to roll the PR back after Canary validation of basic functionality:

Telemetry stands independently of composite logger. 
-	Telemetry may be invoked independent of the logger without any of that showing up in the on-machine log.
-	The logger may write to telemetry but that’s because it’s using that functionality, not because this is the gatekeeper to accessing the telemetry functionality.
-	Additionally the main change was supposed to be changing the telemetry writer transparently to flow out telemetry that’s already set up. (Okay with composite logger utilizing telemetry as it used to in the past in the previous logging mechanism)

Reply:
As shared offline - ^ this is a must for the full PR. Okay for one-off validation and then roll back of the code.